### PR TITLE
feat(form-builder): split build into multiple files

### PR DESCRIPTION
**Pull request by Hyperkid123**

### CODE REVIEW

File: packages/react-form-renderer/rollup.config.js [hunk 1]
```diff
@@ -29,27 +32,41 @@ function onwarn(warning) {
   throw Error(warning.message);
 }
 
-export default [{
+const plugins = [
+  nodeResolve(),
+  babel(babelOptions),
+  commonjs(commonjsOptions),
+  nodeGlobals(), // Wait for https://github.com/cssinjs/jss/pull/893
```
Comment (rvsia) [inline packages/react-form-renderer/rollup.config.js]: what is this comment? :eyes: The mentioned PR has been merged!

File: packages/react-form-renderer/src/components/validator-types.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const validatorTypes = {
+  REQUIRED: 'required-validator',
+  /**
+     * min length of the input value
```
Comment (rvsia) [inline packages/react-form-renderer/src/components/validator-types.js]: what about removing these comments? Sounds pretty redundant to me.

File: packages/react-form-renderer/rollup.config.js (hunk 1 above)
Comment (Hyperkid123) [reply packages/react-form-renderer/rollup.config.js]: Oh right this was due to SSR. We should keep that one. I will just remove the comment

**Comment by Hyperkid123** [comment]
@rvsia renamed the commit

**Pull request merged**